src/core/ext/transport/chttp2/transport/writing.cc

```diff
@@ -715,6 +724,14 @@ grpc_chttp2_begin_write_result grpc_chttp2_begin_write(
               copy_context_fn(s->arena), outbuf_relative_start_pos,
               num_stream_bytes, s->byte_counter, s->write_counter - 1, nullptr);
         }
+        // New way of collecting TCP traces
+        // TODO(yashykt): Should this be protected by a channel arg?
```

Comment:
Is this an actionable TODO? I'm not sure what info you'd need to make that call.

Reply:
I don't think it's actionable right now given that there's no usage. But, if it becomes a concern, I can imagine a channel arg that disables this.

Reply:
removed